### PR TITLE
Use chrome to generate thumbnails

### LINKED ISSUE #236: Replace PhantomJS for creating thumbnail
PhantomJS is no longer supported and does not work correctly.

See #234

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Fixes #236 

## How was this change tested?

Test suite

## Which documentation and/or configurations were updated?

Added `chrome_path` configuration to settings.yml

### CODE REVIEW

**Comment by jcoyne** [comment]
Node has now been updated and I've been able to deploy this to stage.

File: spec/was_seed_preassembly/lib/thumbnail_generator_service_spec.rb [hunk 1]
```diff
@@ -21,20 +16,12 @@
     end
 
     it 'generates jp2 from jpeg thumbnail and pushes to druid_tree content directory', :image_prerequisite do
-      allow(Dor::WASSeed::ThumbnailGeneratorService).to receive(:capture).and_return('')
+      allow(Dor::WASSeed::ThumbnailGeneratorService).to receive(:capture)
```
Comment (ndushay) [inline spec/was_seed_preassembly/lib/thumbnail_generator_service_spec.rb:19]: rando question - this is a step that will go away when we no longer have a thumbnail-service, amirite?

Comment (jcoyne) [reply spec/was_seed_preassembly/lib/thumbnail_generator_service_spec.rb:19]: The ThumbnailGeneratorService is not at all related to https://github.com/sul-dlss/was-thumbnail-service/

Comment (ndushay) [reply spec/was_seed_preassembly/lib/thumbnail_generator_service_spec.rb:19]: ah - i am confusing it with the "update-thumbnail-generator" step for was-seed-dissemination, which presumably will go away when was-thumbnail-server goes away.  Imagine my getting those confused.  🙄

Comment (jcoyne) [reply spec/was_seed_preassembly/lib/thumbnail_generator_service_spec.rb:19]: Yes, it is very confusing.

**Pull request merged**